configs/tsconfig.node-tests.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "./tsconfig.node.json",
+  "compilerOptions": {
+    "composite": true,
+    "emitDeclarationOnly": true,
+    "declarationMap": false
+  },
+  "include": ["${configDir}/src/**/*.test.ts"],
+  "exclude": []
+  /*
+  "references": [
+    {
+      "path": "./tsconfig.json"
+    }
+  ]
+  */
+}
```

Comment:
As a drive-by refactor, I'm creating this tsconfig, which I intent to share from other packages of the repo.